### PR TITLE
chore(): update tests to use local files

### PR DESCRIPTION
**Pull request by gunhansancar**
chore(): update tests to use local files

### CODE REVIEW

File: app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt [hunk 1]
```diff
@@ -31,7 +32,7 @@ import tv.superawesome.demoapp.util.waitUntil
 @SmallTest
 class BannerUITest {
     @get:Rule
-    var wireMockRule = WireMockRule(8080)
+    var wireMockRule = WireMockRule(wireMockConfig().port(8080), false)
```
Comment (thomas-james-1986) [inline app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt:35]: What does the false boolean control?

**Review by thomas-james-1986 — APPROVED** [review]
Looks good, just a query on the WireMock boolean

File: app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt (hunk 1 above)
Comment (gunhansancar) [reply app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt:35]: It is to control `failOnUnmatchedRequest` checks.
That internal check for the request matching was giving false-positive failed tests when I run the whole suite of tests. 
Our verify checks for the requests still work.

Comment (thomas-james-1986) [reply app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt:35]: Nice 👍

**Pull request merged**